readthedocs/api/v3/serializers.py

```diff
@@ -387,7 +382,7 @@ def get_aliases(self, obj):
             if obj.slug == LATEST:
                 alias_version = obj.project.get_original_latest_version()
             if alias_version and alias_version.active:
-                return [self.version_serializer(alias_version).data]
+                return [self.__class__(alias_version).data]
```

Comment:
Can we just call VersionSerializer directly? Is this because of subclasses? It could definitely use a comment here why this is the approach.